evm/src/executor/inspector/cheatcodes/util.rs

```diff
@@ -222,7 +244,15 @@ pub fn apply<DB: Database>(
 ) -> Option<Result> {
     Some(match call {
         HEVMCalls::Addr(inner) => addr(inner.0),
-        HEVMCalls::Sign(inner) => sign(inner.0, inner.1.into(), data.env.cfg.chain_id.into()),
+        HEVMCalls::Sign0(inner) => sign(inner.0, inner.1.into(), data.env.cfg.chain_id.into()),
+        HEVMCalls::CreateWallet0(inner) => {
+            create_wallet(U256::from(keccak256(&inner.0)), Some(inner.0.clone()), state)
+        }
+        HEVMCalls::CreateWallet1(inner) => create_wallet(inner.0, None, state),
+        HEVMCalls::CreateWallet2(inner) => create_wallet(inner.0, Some(inner.1.clone()), state),
+        HEVMCalls::Sign1(inner) => {
+            sign(inner.0.private_key, inner.1.into(), data.env.cfg.chain_id.into())
+        }
```

Comment:
Just wanted to compile a quick list to explain the functionality of each changed line here:

- `Sign0`: the same as the old `Sign`. Just needed to be renamed for new `function sign(Wallet)`
- `CreateWallet0`: `function createWallet(string)` to derive private key and label wallet with `string`
- `CreateWallet1`: `function createWallet(uint256)` create new wallet with private key `uint256` _(no label)_
- `CreateWallet2`: `function createWallet(uint256,string)` create new wallet with private key `uint256` and label with `string`
- `Sign1`: `function sign(Wallet,bytes32)` but uses wallet's private key

Reply:
We can add these as comments, they're great!